analytics/analytics.go

```diff
@@ -22,6 +22,13 @@ const (
 	PredefinedTagGraphAnalytics = "tyk-graph-analytics"
 )
 
+const (
+	operationTypeQuery        = "query"
+	operationTypeMutation     = "mutation"
+	operationTypeSubscription = "subscription"
+	operationTypeUnknown      = "unknown"
+)
```

Comment:
It might make sense for pump to have its own version, but I just wanted to let you know that this exact thing exists in graphql-go-tools.

Reply:
Oh, didn't know about that, i'll look through the go-tools and use it's version, it's better if there is some standard followed.